README.md

```diff
@@ -44,6 +44,9 @@ Install the mapview garden module using the `garden` tool:
 ```
 garden install mapview
 ```
+For Python 3 if you use markers you will get syntax error which can be resolved as follows-
```

Comment:
Minor: with the dash `-` at the end?

Reply:
@AndreMiras 
Thank You for the quick reply.Yes it would have been much better to fix it instead of just documenting a workaround.But I just thought we can have it until a real fix comes.
I also understand that PR should be given to the one who fix it.
So we can close this thread and I'll post the workaround on stackoverflow for reference.

Reply:
Sure, I'm investigating how to unit test the project properly first. I have few ideas already, I'll see what fit best our needs. Once the bug is reproduced in the unit tests I'll pull request the fix